release-managers.md

```diff
@@ -35,11 +35,33 @@ The responsibilities of each role are described below.
 
 ## Release Managers
 
+**Note:** The documentation might refer to the Patch Release Team and the
+Branch Management role. Those two roles were consolidated into the
+Release Managers role.
+
+Minimum requirements for Release Managers and Release Manager Associates are:
+
+- Familiarity with basic Unix commands and able to debug shell scripts.
+- Familiarity with branched source code workflows via `git` and associated
+  `git` command line invocations.
+- General knowledge of Google Cloud (Cloud Build and Cloud Storage).
+- Open to seeking help and communicating clearly.
+- [Kubernetes Community Membership](https://github.com/kubernetes/community/blob/master/community-membership.md#member)
+
 Release Managers are responsible for:
 
-- Coordinating patch releases (`x.y.z`, where `z` > 0) of Kubernetes
-- Minor releases (`x.y.z`, where `z` = 0) of Kubernetes, working with the
-  [Release Team](/release-team/README.md) through each release cycle
+- Coordinating and cutting Kubernetes releases:
+  - Patch releases (`x.y.z`, where `z` > 0)
+  - Minor releases (`x.y.z`, where `z` = 0)
+  - Pre-releases (alpha, beta, and release candidates)
+  - Working with the [Release Team](/release-team/README.md) through each
+  release cycle
+  - Setting the [schedule and cadence for patch releases](/releases/patch-releases.md)
+- Maintaining the release branches:
+  - Running `krel ff` when applicable
```

Comment:
I think we can remove this line, it's not intended to use `krel ff` any more.

Reply:
same as sascha said. 
lgtm

Reply:
Done.